_episodes/05-starting-jekyll.md

```diff
@@ -21,4 +21,57 @@ keypoints:
 - use it again in another page
 - change the value of the global parameter and study the effect
 
+
+> ## Creating your own variable
+>
+> Let's practice making and using variables.
+> Think of a variable you may want to use in your `index.md` page.
+> Some possible options include: your name, the institution you work at, the PI of your lab group.
+> Create the variable and reference it in your `index.md` and save/commit that to your repository.
+> What did you add to your `index.md` to create this variable?
+> Where did you add the line above to your `index.md`?
+> How did you reference that variable?
+>
+> > ## Solution
+> >
+> > We will create the `pi` variable by adding the following line to the yaml header.
+> > ~~~
+> > pi: "Ada Lovelace"
```

Comment:
Perhaps reinforce the need for the `---` guards to delimit the header.

Reply:
And since we are mentioning YAML for the first time, make a note about the use of no-quotes, single-quotes and double-quotes. [TIL that there's a difference](https://www.yaml.info/learn/quote.html)

Reply:
> ... [TIL that there's a difference](https://www.yaml.info/learn/quote.html)  

is really helpful. To understand `# mnemonic: '>' is a folded '|'` [this link](https://stackoverflow.com/a/21699210/8935243) helped me.

Reply:
I think it would be more manageable to introduce just enough YAML for learners to make progress with the lesson, but to make sure they know there's more to learn and where to find it. Perhaps a callout to learn more about YAML? In #29 @anenadic included a callout pointing learners to online YAML linters, which is also really useful.

Or do you think the quoting differences are important/hazardous enough that they can't be skipped over here?

Reply:
I don't think it's a major issue unless special characters are involved, which I would qualify as non-introductory.
Perhaps more of a note to self/us that we should pick one and be consistent throughout the materials.

Reply:
Agree we should be consistent.  And the link to examples is really helpful.  Since we don't strictly need the quotes do you think we should remove them here?

Reply:
Also, maybe we should have an issue to discuss what we will consistently use? and keep this conversation in our minds?

Reply:
I'll add this to the agenda for our October sprint